pulsar/client_impl.go

```diff
@@ -30,10 +30,12 @@ import (
 )
 
 const (
-	defaultConnectionTimeout = 10 * time.Second
-	defaultOperationTimeout  = 30 * time.Second
-	defaultKeepAliveInterval = 30 * time.Second
-	defaultMemoryLimitBytes  = 64 * 1024 * 1024
+	defaultConnectionTimeout     = 10 * time.Second
+	defaultOperationTimeout      = 30 * time.Second
+	defaultKeepAliveInterval     = 30 * time.Second
+	defaultMemoryLimitBytes      = 64 * 1024 * 1024
+	defaultConnectionMaxIdleTime = 60 * time.Second
```

Comment:
Refer [java impl](https://github.com/apache/pulsar/blob/0f72a822dd8aa21fa3d86bb6f62ea4482fc3b907/pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java#L128-L133), the defaults connection max idle time should be 180s

Reply:
Thanks. Fixed it.